hack/deploy/configure-values.sh

```diff
@@ -51,7 +60,7 @@ KARPENTER_USER_ASSIGNED_CLIENT_ID=$(az identity show --resource-group "${AZURE_R
 
 export CLUSTER_NAME AZURE_LOCATION AZURE_RESOURCE_GROUP_MC KARPENTER_SERVICE_ACCOUNT_NAME \
```

Comment:
Did you test this on both styles?

Reply:
you mean custom vnet + managed vnet? Or configure-values.sh doing everything vs passing values in?

Reply:
Tested both, they work fine, although I am wondering if we could have some basic test harness for different AKS_JSONs for this type of thing.

Reply:
Not going to solve here but worth considering